src/C-DEngine/C-DEngines/C-DNMIService/cde_NMIAPI.cs

```diff
@@ -2448,7 +2448,7 @@ public static TheFormInfo CreateEngineForms(ICDEThing pBaseThing, Guid pID, stri
         /// <param name="pFlags"></param>
         /// <param name="pACL"></param>
         /// <returns></returns>
-        public static cdeConcurrentDictionary<string,TheFieldInfo> AddCommonTableColumns(TheThing MyBaseThing, TheFormInfo pTargetForm, string pDeviceTypeOptions=null,string pDeviceTypeDefault=null,bool AddAddress=true, bool AddDetailsButton = true, int pFldOrder = 1000, int pFlags = 0xA2, int pACL = 0x80)
+        public static cdeConcurrentDictionary<string,TheFieldInfo> AddCommonTableColumns(TheThing MyBaseThing, TheFormInfo pTargetForm, string pDeviceTypeOptions=null,string pDeviceTypeDefault=null,bool AddAddress=true, bool AddDetailsButton = true, int pFldOrder = 1000, int pFlags = 0xA2, int pACL = 0x80, bool AddNodeName = false)
```

Comment:
you will have to create an override for this function as this is used in MANY plugins and they will all fail even though they dont use that parameter.

Reply:
Good catch. Did think about that but then forgot. Fixed now.